pkg/prm/backends/docker_test.go

```diff
@@ -0,0 +1,82 @@
+package backends
```

Comment:
move tests out of the package they are testing so: `backends_test`
This ensures that the package works as a reusable component